specials/PF_FormEdit.php

```diff
@@ -49,15 +49,17 @@ function printAltFormsList( $alt_forms, $target_name ) {
 		$fe = PFUtils::getSpecialPage( 'FormEdit' );
 		$fe_url = $fe->getPageTitle()->getFullURL();
 		$i = 0;
-		foreach ( $alt_forms as $alt_form ) {
-			if ( $i++ > 0 ) {
-				$text .= ', ';
+		if ( is_iterable( $alt_forms ) ) {
```

Comment:
do we know what are the cases when it's not iterable?

```
$alt_forms = $this->getRequest()->getArray( 'alt_form' );
```
this doesn't guarantee to get an array?